### DISCUSSION TITLE
软件打开白屏

### DISCUSSION THREAD

**Discussion by chensosoooo** [localsend/localsend#553] [L-简体中文]
请问一下打开软件，没有任何界面是缺少什么插件吗？win10 64位 我另一台电脑可以使用。
May I ask, when I open the software and there is no interface, is it due to missing plugins? Windows 10 64-bit. The software works on my other computer.
<img width="467" alt="3cb4d8fc4dace24889f50ab4b3f45b6" src="https://github.com/localsend/localsend/assets/135323999/c53ad72a-fcda-4e85-94dc-2ff8654d443a">

**Comment by TheCrawlingChaos** [comment]
遇到了同样问题，发现是N卡控制面板3D全局设置为了高性能NVIDIA处理器 。(NVIDIA控制面板--管理3D设置-全局设置-高性能NVIDIA)
解决方法:N卡控制面板里把Localsend改为集成图形处理器

**Reply by xiaochopin** [reply]
我用的amd核显，有相同问题
